Sources/SwiftDocC/Model/DocumentationMarkup.swift

```diff
@@ -73,6 +73,14 @@ struct DocumentationMarkup {
         case end
     }
     
+    /// Directives which are removed from the markdown content after being parsed.
+    private static let directivesRemovedFromContent = [
+        Comment.directiveName,
+        Metadata.directiveName,
+        Options.directiveName,
+        Redirect.directiveName,
+    ]
```

Comment:
I think _maybe_ that we should also add `DeprecationSummary` to this list (see rdar://79719308)—but that could fit better in a separate PR.

Reply:
Sure sounds good. I can take that Radar.